### PR TITLE
Added a limit of 10 MB as max single file size

### PR DESCRIPTION
**Pull request by NLuffy**

### CODE REVIEW

File: src/main/java/com/example/simplefileshare/simplefileshare/controller/FileController.java [hunk 1]
```diff
@@ -33,4 +33,9 @@ public ResponseEntity<BaseResponse<String>> upload(@RequestParam("file") Multipa
             throw ErrorUtils.createApiError(HttpStatus.INTERNAL_SERVER_ERROR, HttpStatus.INTERNAL_SERVER_ERROR.getReasonPhrase(), "Some file error occurred", e.getStackTrace());
         }
     }
+
+    private void CheckFileSanity(MultipartFile file) throws Exception {
```
Comment (shashank-vats) [inline src/main/java/com/example/simplefileshare/simplefileshare/controller/FileController.java:37]: Please make it start with small case letter

Comment (NLuffy) [reply src/main/java/com/example/simplefileshare/simplefileshare/controller/FileController.java:37]: There is no harm in this format also for private methods.

Comment (shashank-vats) [reply src/main/java/com/example/simplefileshare/simplefileshare/controller/FileController.java:37]: Please check this [link](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf ). 
image

**Review by shashank-vats — APPROVED** [review]

File: src/main/java/com/example/simplefileshare/simplefileshare/controller/FileController.java (hunk 1 above)
Comment (NLuffy) [reply src/main/java/com/example/simplefileshare/simplefileshare/controller/FileController.java:37]: Read the conventions on the net also but couldnt find a reason for it. FYI already modified it in the next PR.

**Pull request merged**